src/main/java/racing/domain/RacingCars.java

```diff
@@ -0,0 +1,59 @@
+package racing.domain;
+
+import racing.domain.RacingCar;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+    private static final String NAME_DELIMITER = ",";
+    private final List<RacingCar> racingCarList;
+
+    public RacingCars(String names) {
+        if (isNullOrBlank(names)) {
+            throw new IllegalArgumentException();
+        }
+
+        this.racingCarList = Arrays.stream(toNameArray(names))
+                .map(RacingCar::new)
+                .collect(Collectors.toList());
+    }
+
+    public RacingCars(List<RacingCar> racingCarList) {
+        this.racingCarList = racingCarList;
+    }
+
+    public List<RacingCar> getRacingCarList() {
+        return racingCarList;
+    }
+
+    public int size() {
+        return racingCarList.size();
+    }
+
+    public int getMaxPosition() {
```

Comment:
```suggestion
    private int getMaxPosition() {
```

테스트를 위해 public으로 오픈하는것이 안티패턴은 아니라고 생각합니다.
하지만, 내부에서만 사용하는 함수라면 최대한 값을 은닉시키고 `getWinnerNames` 검증으로 충분할것 같은데 이부분에 대해 어떻게 생각하시나요? 😄